test/test_testing.py

```diff
@@ -1147,5 +1147,69 @@ def test_mismatching_device_no_check(self, devices):
 instantiate_device_type_tests(TestAssertsMultiDevice, globals(), only_for="cuda")
 
 
+class TestAssertsSparse(TestCase):
```

Comment:
No csr tests?

Reply:
The PR is currently lacking sparse csr support. I think it should be provided within this PR.